tests/schema/tests.py

```diff
@@ -540,18 +540,22 @@ def test_alter_o2o_to_fk(self):
         # Ensure the field is right to begin with
         columns = self.column_classes(BookWithO2O)
         self.assertEqual(columns['author_id'][0], "IntegerField")
+        # Ensure the field is unique to begin with
+        author = Author.objects.create(name="Joe")
+        BookWithO2O.objects.create(author=author, title="Django 1", pub_date=datetime.datetime.now())
+        self.assertRaises(
```

Comment:
I just copied the code over from the unique test case to see if the previous patch worked at all on Oracle. Cleaning up now.